libs/blocks/card/card.js

```diff
@@ -94,25 +109,33 @@ const addFooter = (links, container) => {
 };
 
 const init = (el) => {
+  const { miloLibs, codeRoot } = getConfig();
+  const base = miloLibs || codeRoot;
+  loadStyle(`${base}/deps/caas.css`);
+
   const section = el.closest('.section');
+  section.classList.add('milo-card-section');
   const row = el.querySelector(':scope > div');
   const picture = el.querySelector('picture');
   const links = el.querySelectorAll('a');
   const styles = Array.from(el.classList);
   const cardType = getCardType(styles);
-  const version = new URL(document.location.href)?.searchParams?.get('caasver') || 'latest';
   let card = el;
 
-  loadStyle(`https://www.adobe.com/special/chimera/${version}/dist/dexter/app.min.css`);
   addWrapper(el, section, cardType);
 
   if (cardType === HALF_HEIGHT) {
-    card = document.createElement('a');
-    card.href = links[0]?.href || '';
-    card.className = el.className;
-
-    el.insertAdjacentElement('beforebegin', card);
-    links[0]?.parentElement?.remove();
+    const [link] = links;
+
+    if (link) {
+      card = link;
+      el.prepend(link);
+    } else {
+      card = document.createElement('a');
+      card.href = link?.href || '';
+      el.prepend(card);
+      link?.parentElement?.remove();
+    }
```

Comment:
Lines 135 & 137 reference `link` which will be undefined.  Should that be in the `if (link)` block?

Also `el.prepend(card) can be moved to below this if/else

Reply:
Good catch, not sure how that snuck in there